docs/sources/operators-guide/configuring/configuring-shuffle-sharding/index.md

```diff
@@ -116,21 +121,21 @@ Keeping ingesters shuffle sharding enabled only on the write path does not lead
 If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and you want to enable it for the ingesters, use the following rollout strategy to avoid missing querying for any series currently in the ingesters:
 
 1. Enable ingesters shuffle sharding on the write path.
```

Comment:
Mention to disable shuffle sharding on read path, and which flag to use. It's not clear now.

Reply:
Let me think loudly. We start from the assumption shuffle sharding is disabled beforehand, but `-querier.shuffle-sharding-ingesters-enabled` default is `true`, which means as soon as you "enable" shuffle sharding setting the ingesters shard size (for all tenants or just one) then shuffle sharding will be immediately enabled on read path too. We don't want it. We want you to set shard size and `-querier.shuffle-sharding-ingesters-enabled=false` at first, then wait some time, then switch `-querier.shuffle-sharding-ingesters-enabled=true`.

Makes sense to you?

Reply:
Yes, thank you. I'll work on something!